### DISCUSSION TITLE
epaper WS 3.7 - 1bit driver

### DISCUSSION THREAD

**Discussion by Kvasir88** [peterhinch/micropython-nano-gui#40] [General]
Dear Peter,
thanks for your great nano-gui.
I ordered a Waveshare epaper 3.7" and tried to adapt your driver for epaper 4.2.
Not really knowing what why how.... i took the two original driver 3.7 and 4.2 and tried to understand what you did.
So far, the luts are changed to binary and commands has been adapted.
The original 3.7 driver has some code like load lut i dont know what to do with.
Do you please have a hint how to solve this?

PS: this is my first post on github, a big new world. Since now i tried to learn by searching, reading, watch tutorials..
Best regards,
Daniel

**Comment by peterhinch** [comment]
It's hard to advise on a display I don't possess. In general my approach is to take the C driver and translate to Python, but it's a fairly involved process and not amenable to a simple set of suggestions. I don't feel keen on buying one of these displays when I already support one of fairly similar size.
